source/tool/chipsec/hal/acpi_tables.py

```diff
@@ -584,3 +584,27 @@ def __str__( self ):
 ACPI Table Entries:
 %s
 """ %( ''.join( ['0x%016X\n' % addr for addr in self.Entries]))
+
+########################################################################################################
+#
+# FACP Table
+#
+########################################################################################################
+
+class FACP (ACPI_TABLE):
+  def __init__( self ):
+    self.DSDT = None

```

Comment:
Attributes should be lower-case. I know Chipsec does not have much consistency there, but let's move in the correct direction.